src/elements/Contact.jsx

```diff
@@ -135,17 +123,25 @@ class Contact extends Component {
             </div>
           </div>
         </div> */}
-        {/* End Brand Area */}
+      {/* End Brand Area */}
 
-        {/* Start Back To Top */}
-        <div className="backto-top">
-          <ScrollToTop showUnder={160}>
-            <FiChevronUp />
-          </ScrollToTop>
-        </div>
-        {/* End Back To Top */}
-      </React.Fragment>
-    )
-  }
+      {/* Start Back To Top */}
+      <div className="backto-top">
+        <ScrollToTop showUnder={160}>
+          <FiChevronUp />
+        </ScrollToTop>
+      </div>
+      {/* End Back To Top */}
+    </>
+  )
+}
+
+Contact.defaultProps = {
+  center: {
+    lat: -34.397,
+    lng: 150.644,
+  },
+  zoom: 8,
 }
```

Comment:
I think it would be better to destructure our props and assign the default values there. The beta React docs only mention using defaultProps for class components.

https://beta.reactjs.org/learn/passing-props-to-a-component#specifying-a-default-value-for-a-prop